extensions/sql-migration/package.json

```diff
@@ -89,6 +89,8 @@
   },
   "dependencies": {
     "@microsoft/ads-extension-telemetry": "^1.1.3",
+    "@types/uuid": "^8.3.0",
```

Comment:
`@types` packages should always go in devDependencies, they won't have any runtime components.

Reply:
@aasimkhan30 ping